climpred/tests/test_hindcastEnsemble_class.py

```diff
@@ -165,5 +165,11 @@ def test_verify_m2o_reference(hindcast_hist_obs_1d):
     hindcast.verify(metric='mse', comparison='m2o', dim='init', reference='persistence')
     # probabilistic
     hindcast.verify(
-        metric='crps', comparison='m2o', dim='member', reference='historical'
+        metric='crps', comparison='m2o', reference='historical', dim='member'
+    )
+
+
+def test_bootstrap(hindcast_hist_obs_1d):
```

Comment:
Add test for when you have multiple observational datasets.

Reply:
dismissed in #429